src/test/java/SalaryCalculatorTest.java

```diff
@@ -1,42 +1,63 @@
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.internal.configuration.injection.scanner.MockScanner;
 
 import java.io.IOException;
 
 public class SalaryCalculatorTest {
 
-    private Main main;
+    private SalaryCalculator salaryCalculator;
+
+    SalaryCalculator salaryCalculator0 = new SalaryCalculator("Andriy", "Vareg", 200, 172, 61, 100);
+    SalaryCalculator salaryCalculator1 = new SalaryCalculator("Vasyl", "Arial", 250, 172, 52, 125);
+    SalaryCalculator salaryCalculator2 = new SalaryCalculator("Oleh", "Serif", 50, 172, 9, 25);
+    SalaryCalculator salaryCalculator3 = new SalaryCalculator("Karl", "Aref", 150, 172, 12, 75);
+    SalaryCalculator salaryCalculator4 = new SalaryCalculator("Karolina", "Vagen", 100, 172, 0, 50);
+    SalaryCalculator salaryCalculator5 = new SalaryCalculator("Pavlo", "Agen", 300, 172, 150, 150);
 
 
     @Before
     public void setup() {
 
-        main = new Main();
+        salaryCalculator = new SalaryCalculator();
     }
 
 
     @Test
     public void salaryTest() {
 
-        int actual = main.salary(main.salaryCalculator0.getSalaryPerHour(), main.salaryCalculator0.getHoursWork(), main.salaryCalculator0.getHoursHospital(),main.salaryCalculator0.getHourHospitalSalary());
+        int actual = salaryCalculator.salary(salaryCalculator0.getSalaryPerHour(), salaryCalculator0.getHoursWork(), salaryCalculator0.getHoursHospital(), salaryCalculator0.getHourHospitalSalary());
```

Comment:
You need to pass just some numbers instead of getting them from `salaryCalculator0`.
For instance:

```
 int actual = salaryCalculator.salary(21, 24, 10, 2);
```